src/components/Login.js

```diff
@@ -1,83 +1,87 @@
 import React, { useState, useEffect } from "react";
 import Form from "react-bootstrap/Form";
 import Button from "react-bootstrap/Button";
-import { createPatient } from "../firebase";
+import {  validateParticipant } from "../firebase";
 import { jsPsych } from "jspsych-react";
 import "../App.css";
 import "bootstrap/dist/css/bootstrap.css";
 import "@fortawesome/fontawesome-free/css/all.css";
 
-function Login({ipcRenderer, callBack}) {
+function Login({onLogin, participant_id, study_id, validation}) {
```

Comment:
would call the props something more distinct than the entered values - the nuance between the snake and camel case versions will be easy to miss (maybe `envParticipantId` or something)